lochness/redcap/__init__.py

```diff
@@ -640,5 +646,7 @@ def update_study_metadata(subject, content: List[dict]) -> None:
             Path(Lochness['phoenix_root']) / 'PROTECTED').glob('*/raw/*'):
         subject = subject_path.name
         print(subject)
-        # get_run_sheets_for_datatypes(api_url, api_key, subject, id_field, json_path)
```

Comment:
Why was this line inactive so far? My understanding is that we have been already pulling JSONs.

Reply:
This line under `if __name__ == '__main__':` is for testing purpose. The actual line for running `get_run_sheets_for_datatypes` is under `lochness.redcap.sync` function.